src/models/FormPublish.ts

```diff
@@ -43,6 +46,8 @@ export const initialValues: Partial<MetadataPublishFormDataset> = {
   description: '',
   timeout: 'Forever',
   access: '',
+  provider: '',
   termsAndConditions: false,
-  tags: ''
+  tags: '',
+  customProvider: ''
```

Comment:
who do we collect a `provider` & a `customProvider` here? This is one field and we should capture different values for it, not every value in its own key?

Reply:
I've now reduced it to just one field